src/bFS/BFS.java

```diff
@@ -0,0 +1,97 @@
+package bFS;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class BFS {
+	private ArrayList<Node> list;
+	private ArrayDeque<Node> deque;
+	private Node root;
+
+	public BFS() {
+		list = new ArrayList<Node>();
+		deque = new ArrayDeque<Node>();
+		root = null;
+	}
+
+	public void addNode(Node n) {
+		list.add(n);
+	}
+
+	public void setRoot(int r) {
+		root = list.get(r-1);
+	}
+
+	public void setChild(int x, int y) {
+		list.get(x-1).addChild(list.get(y-1));
+	}
```

Comment:
avoid using one-liner methods, especially when you are only gonna call them once or twice in the main method. You can easily replace Line 90 with Line 25 and replace 86-87 with Line 29.